rules/react.js

```diff
@@ -41,7 +42,7 @@ module.exports = {
     "react/self-closing-comp": ["error"],
     "react/sort-comp": ["off"],
     "react/prefer-stateless-function": ["off"],
-    "react/destructuring-assignment": ["off"],
+    "react/destructuring-assignment": ["error", "always"],
```

Comment:
```suggestion
    "react/destructuring-assignment": ["error"],
```

default value is `always`

Reply:
You are right, I was making test on it, that's why I let this param :wink: